projects/rocprofiler-systems/examples/videodecode/video_demuxer.h

```diff
@@ -570,9 +570,24 @@ class VideoDemuxer
              !strcmp(av_fmt_input_ctx_->iformat->long_name, "FLV (Flash Video)") ||
              !strcmp(av_fmt_input_ctx_->iformat->long_name, "Matroska / WebM"));
 
+        // For latest version of FFMPeg, read_seek and read_seek2 are not exposed in
+        // AVInputFormat
+#if USE_AVCODEC_GREATER_THAN_58_134
+        // Check if the input stream is seekable for FFmpeg >= 58.134
+        if((av_fmt_input_ctx_->iformat->flags & AVFMT_NO_BYTE_SEEK) == 0 &&
+           av_fmt_input_ctx_->pb && av_fmt_input_ctx_->pb->seekable)
+        {
+            is_seekable_ = true;
+        }
+        else
+        {
+            is_seekable_ = false;
+        }
```

Comment:
You don't need if/else here

```suggestion
        is_seekable_ = (av_fmt_input_ctx_->iformat->flags & AVFMT_NO_BYTE_SEEK) == 0 &&
               av_fmt_input_ctx_->pb && av_fmt_input_ctx_->pb->seekable;
```

Reply:
Yes. Completely agree! 
Resolved.